lib/string_reverse.rb

```diff
@@ -1,4 +1,18 @@
 # A method to reverse a string in place.
 def string_reverse(my_string)
-  my_string << "not implemented"
+  return nil if my_string == nil
+
+  count = 0
+  max_count = my_string.length - 1
+
+  return false if my_string[count] == nil
+
+  while count <= max_count

```

Comment:
Minor: As a minor optimization, you could make this condition `while count < max_count` and the function will still work.